cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/UserTasksParameters.java

```diff
@@ -29,10 +34,30 @@ public UserTasksParameters(HttpServletRequest request) {
   @Override
   protected void initParameters() throws UnsupportedEncodingException {
     super.initParameters();
-    _userTaskIds = ParameterUtils.userTaskIds(_request);
+    _taskIds = ParameterUtils.taskIds(_request);
+    _clientIds = ParameterUtils.clientIds(_request);
+    _endPoints = ParameterUtils.endPoints(_request);
+    _types = ParameterUtils.types(_request);
+    _entries = ParameterUtils.entries(_request);
   }
 
   public Set<UUID> userTaskIds() {
-    return _userTaskIds;
+    return _taskIds;
+  }
+
+  public Set<String> clientIds() {
+    return _clientIds;
+  }
+
+  public Set<EndPoint> endPoints() {
+    return _endPoints;
+  }
+
+  public Set<UserTaskManager.TaskState> taskStates() {
+    return _types;
```

Comment:
Bad naming! the getter is `taskStates`, but it returns `_types`. The getter should be named `types()`.